src/it/scala/OmegaRepositoryISpec.scala

```diff
@@ -121,4 +121,197 @@ class OmegaRepositoryISpec
     }
   }
 
+  "Get Record Concept Entity" - {
+    "must return a List with one item" in {
+      when(mockConfig.sparqlEndpoint).thenReturn("http://localhost:8080/rdf4j-server/repositories/PACT")
```

Comment:
Should the URL `http://localhost:8080/rdf4j-server/repositories/PACT` or parts of it be moved into constants. It seems a bit strange that this repeats through the test class.

Reply:
Moved into a constant.